huggingface_pipelines/preprocessing.py

```diff
@@ -0,0 +1,201 @@
+from transformers import WhisperProcessor, WhisperFeatureExtractor
+import numpy as np
+import spacy
```

Comment:
you need to add spacy as depedancy at pyproject file. Same for transformers I suppose.